docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,59 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# [RFC] Title here
+
+> Delete lines starting with ">". Remove sections that are not applicable.
+
+## Overview
+
+> If someone only reads this, what do you want them to know?
+
+## Goals and Non-Goals
```

Comment:
I would recommend having `Background & Motivation` before that point.

IMO it seems more natural :shrug:

Reply:
Done